lexicons/com/atproto/identity/defs.json

```diff
@@ -0,0 +1,26 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.identity.defs",
+  "defs": {
+    "atprotoIdentity": {
```

Comment:
`com.atproto.identity.defs#atprotoIdentity` has a bit of a weird ring to me. like it already has `atproto.identity` in the NSID & then just repeats itself. Maybe `#identityDetails` or `#identityDetailed` or something to make it more descriptive of what's in there.

But also this is only used in refs (not open unions) and lots of our `defs` schema fragments end up having weird/long names. So probably fine.

I will say it reminds me a bit of `com.atproto.repo.strongRef` in being such a foundational concept. We hoisted that one up to a top level NSID. I kinda regret that because it's the only one of its kind, but it _could_ make sense to do something similar here?

Not really sure, just airing some ideas, this isn't a huge hangup for me.

Reply:
`com.atproto.identity.defs#identity` felt a bit weird to me, but is also an option. Looking at it again, having `atproto` as a prefix is kind of weird, given this is in the `com.atproto.*` namespace.

I think I lean away from top-level NSIDs like `com.atproto.repo.strongRef` for this sort of thing. Especially this one, i'm not sure it will actually get re-used often? Passing around the full DID Document as `unknown` isn't super ergonomic.

(for `strongRef`, in the mid-term i'm hoping we can extend the `at://` syntax to accommodate the strongRef use-case, and not have such a basic building-block require a Lexicon reference)